content/books/golden-age-of-intelligence/chapter-03-what-the-ancients-knew.md

```
@@ -0,0 +1,267 @@
+# Chapter 3 — What the Ancients Knew
+
+> *"Confine yourself to the present."*
+> — Marcus Aurelius, *Meditations*
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)